controllers/update.go

```diff
@@ -84,6 +85,19 @@ func (r *DevfileRegistryReconciler) updateDeployment(ctx context.Context, cr *re
 			needsUpdating = true
 		}
 	}
+	headlessStatus := dep.Spec.Template.Spec.Containers[0].Env
```

Comment:
Is it possible to rename this var to be more clear that it is the array of all env vars for the container?